dist/pathfora.js

```diff
@@ -948,26 +917,65 @@
         utils.addClass(widget, 'pf-no-img');
       }
 
-      widgetMessage.innerHTML = config.msg;
+      switch(config.type) {
+      case 'sitegate':
+      case 'form':
 
-      if (config.type === 'form') {
-        if (config.nameField === false) {
-          // FIXME Cache
-          widgetForm.removeChild(widgetForm.querySelector('input[name="username"]'));
-        }
-        if (config.titleField === false) {
-          // FIXME Cache
-          widgetForm.removeChild(widgetForm.querySelector('input[name="title"]'));
-        }
-        if (config.emailField === false) {
-          // FIXME Cache
-          widgetForm.removeChild(widgetForm.querySelector('input[name="email"]'));
+        if (config.showSocialLogin === false) {
+          node = widget.querySelector('.pf-social-login');
+
+          if (node.parentNode) {
```

Comment:
probably worth checking that node is not null here. if for some reason `.pf-social-login` doesnt exist this error.